src/app/context/AuthContext.tsx

```diff
@@ -35,27 +36,31 @@ export function AuthProvider({ children }: { children: React.ReactNode }) {
     });
   };
 
-  const getAccountInfo = (accountId?: string) => {
-    const id = accountId || account?.id;
+  const getAccountInfo = () => {
+    const id = account?.id;
     if (!id) return;
     setAccount({ id }); // Clear current info.
-    utils.call("accountInfo").then((response) => {
+    api.getAccountInfo().then((response) => {
       const { alias } = response.info;
       const balance = parseInt(response.balance.balance); // TODO: handle amounts
       setAccount({ id, alias, balance });
     });
   };
 
   useEffect(() => {
-    utils
-      .call("status")
+    api
+      .getStatus()
       .then((response) => {
         if (!response.configured) {
           utils.openPage("welcome.html");
           window.close();
         } else if (response.unlocked) {
-          setAccount(response.currentAccountId);
-          getAccountInfo(response.currentAccountId);
+          setAccount({ id: response.currentAccountId });
+          api.getAccountInfo().then((response) => {
```

Comment:
we need to introduce caching, I think we do this account info call twice now.
(independent of this PR)